test/k8sT/DatapathConfiguration.go

```diff
@@ -325,6 +325,30 @@ var _ = Describe("K8sDatapathConfig", func() {
 			}, DeployCiliumOptionsAndDNS)
 
 			Expect(testPodConnectivityAcrossNodes(kubectl)).Should(BeTrue(), "Connectivity test between nodes failed")
+
+			It("with L7 policy", func() {
```

Comment:
`s/It/Context/`, as AFAIK `It` doesn't allow its body to contain setup routines. From the CI run of this PR:

```
/home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/k8s-1.11-gopath/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:471
You may only call It from within a Describe, Context or When
/home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/k8s-1.11-gopath/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:471
```